platform/docs/docs/configuration/dataSources/dicom-json.md

```diff
@@ -154,6 +154,18 @@ Your public folder should look like this:
 
 ![](../../assets/img/dicom-json-public.png)
 
+:::tip
+It is important to URL encode the `url` query parameter especially if the `url`
```

Comment:
I added these docs for DICOM JSON for an unrelated issue that I recently closed.